khmer/khmer_args.py

```diff
@@ -151,8 +151,7 @@ def create_nodegraph(args, ksize=None, multiplier=1.0):
     if ksize is None:
         ksize = args.ksize
     if ksize > 32:
-        print_error("\n** ERROR: khmer only supports k-mer sizes <= 32.\n")
-        sys.exit(1)
+        raise Exception("** ERROR: khmer only supports k-mer sizes <= 32.")
```

Comment:
Ahh, sorry. I'll switch it back to the old behavior and use pipe redirection to catch the errs.